bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/TemplateUtils.java

```diff
@@ -37,4 +47,94 @@ public static CloudBigtableTableConfiguration BuildImportConfig(ImportOptions op
     }
     return builder.build();
   }
+
+  /**
+   * Provides a request that is constructed using a {@link Scan} object.
+   */
+  private static class RequestWithScanValueProvider
```

Comment:
Scan is an internal implementation detail, it just takes attributes for a ReadRowsRequest and builds it. I don't think RequestWithScan makes sense as a name

Reply:
Yeah, I have doubt about the naming.
When you say "attributes", are you referring to following?
+    private final ValueProvider<String> start;
+    private final ValueProvider<String> stop;
+    private final ValueProvider<Integer> maxVersion;
+    private final ValueProvider<String> filter;

I will just call this class RequestValueProvider then.

Let me know.